packages/next/src/utils.ts

```diff
@@ -2202,6 +2202,7 @@ interface BaseEdgeFunctionInfo {
   page: string;
   wasm?: { filePath: string; name: string }[];
   assets?: { filePath: string; name: string }[];
+  userConfig?: Record<string, unknown>;
```

Comment:
Can we add regions to the type here?

Reply:
Sure

Reply:
I am torn between having `unknown` aka “I don’t trust the input and must parse everything” and having the expected type and trusting the input 🥲

Reply:
In that case, we could throw here if the shape is invalid. Another option is to pass it through to the API and let it respond with 400 if the payload is invalid.

Reply:
The latter is already implemented, so I’ll type it as the API expects